sdk/clientcore/annotation-processor-test/src/main/java/io/clientcore/annotation/processor/test/implementation/TestInterfaceClientImpl.java

```diff
@@ -297,26 +297,6 @@ HttpBinJSON putBodyAndContentLength(@HostParam("uri") String uri,
             @BodyParam(ContentType.APPLICATION_OCTET_STREAM) ByteBuffer body,
             @HeaderParam("Content-Length") long contentLength);
 
-        @HttpRequestInformation(method = HttpMethod.PUT, path = "put", expectedStatusCodes = { 201 })
-        HttpBinJSON putWithUnexpectedResponse(@HostParam("uri") String uri,
-            @BodyParam(ContentType.APPLICATION_OCTET_STREAM) String putBody);
-
-        @HttpRequestInformation(method = HttpMethod.PUT, path = "put", expectedStatusCodes = { 201 })
-        @UnexpectedResponseExceptionDetail(exceptionBodyClass = HttpBinJSON.class)
-        HttpBinJSON putWithUnexpectedResponseAndExceptionType(@HostParam("uri") String uri,
-            @BodyParam(ContentType.APPLICATION_OCTET_STREAM) String putBody);
-
-        @HttpRequestInformation(method = HttpMethod.PUT, path = "put", expectedStatusCodes = { 201 })
-        @UnexpectedResponseExceptionDetail(statusCode = { 200 }, exceptionBodyClass = HttpBinJSON.class)
-        HttpBinJSON putWithUnexpectedResponseAndDeterminedExceptionType(@HostParam("uri") String uri,
-            @BodyParam(ContentType.APPLICATION_OCTET_STREAM) String putBody);
-
-        @HttpRequestInformation(method = HttpMethod.PUT, path = "put", expectedStatusCodes = { 201 })
-        @UnexpectedResponseExceptionDetail(statusCode = { 400 })
-        @UnexpectedResponseExceptionDetail(exceptionBodyClass = HttpBinJSON.class)
-        HttpBinJSON putWithUnexpectedResponseAndFallthroughExceptionType(@HostParam("uri") String uri,
```

Comment:
Were these APIs removed or moved? They seem to test valid cases where different response codes correlate to different exception types.

Reply:
I explained it poorly as type is too vague.

Correct, we don't support different exception types, such as `NotModifiedException extends HttpResponseException`.

I was more talking on how we still allow different exception body types to be configured, which isn't handled.